src/main/scala/de/upb/cs/swt/delphi/instanceregistry/connection/Server.scala

```diff
@@ -40,70 +40,126 @@ class Server (handler: RequestHandler) extends HttpApp
       apiRoutes
     }
   }
+
   //Routes that map http endpoints to methods in this object
-  def apiRoutes : server.Route =
-      /****************BASIC OPERATIONS****************/
-      path("register") {entity(as[String]) { jsonString => register(jsonString) }} ~
-      path("deregister") { deregister() } ~
-      path("instances") { fetchInstancesOfType() } ~
-      path("instance") { retrieveInstance() } ~
-      path("numberOfInstances") { numberOfInstances() } ~
-      path("matchingInstance") { matchingInstance()} ~
-      path("matchingResult") { matchInstance()} ~
-      path("eventList") { eventList()} ~
-      path("linksFrom") { linksFrom()} ~
-      path("linksTo") { linksTo()} ~
-      path("network") { network()} ~
-      path("addLabel") { addLabel()} ~
-      /****************DOCKER OPERATIONS****************/
-      path("deploy") { deployContainer()} ~
-      path("reportStart") { reportStart()} ~
-      path("reportStop") { reportStop()} ~
-      path("reportFailure") { reportFailure()} ~
-      path("pause") { pause()} ~
-      path("resume") { resume()} ~
-      path("stop") { stop()} ~
-      path("start") { start()} ~
-      path("delete") { deleteContainer()} ~
-      path("assignInstance") { assignInstance()} ~
-      path("command") { runCommandInContainer()} ~
-      path("logs") { retrieveLogs()} ~
-      path("attach") { streamLogs()} ~
```

Comment:
The endpoints ```/logs``` and ```/attach``` have been introduced *after* the redesign of the API and do not have any documentation yet. Also i deliberately removed them from the ```apiRoutes``` variable when i merged develop, because they do not follow the new design yet (see #82 for reference). 
Please add the  paths ```GET /instances/<id>/logs``` with *optional query parameter* ```StdErr: Boolean``` and ``` /instances/<id>/attach``` (this is a websocket streaming endpoint, not sure how this will be documented in swagger. If you don't find a viable solution, just skip the documentation for this endpoint for now) with *optional query parameter* ```StdErr: Boolean``` to the documentation and to the ```apiRoutes``` variable. The functionalities are fully implemented, the respective methods in the ```Server.scala``` are named ```retreiveLogs()``` and ```streamLogs()```.

Reply:
Has been done in latest commit.